toc/working-groups/service-management/working-agreements/2021-03-09-reviewer-workflows.md

```diff
@@ -0,0 +1,85 @@
+# Reviewer Workflows for Issues and PRs
+
+**Date of Forum to be Discussed:** Thursday 10 March, 2022 11 am UTC / 4:30 pm IST
+See the community calendar for more details.
+
+**Author:** Marcela Campo - @pivotal-marcela-campo
+
+## Problem
+Our working group has many different projects maintained by disjointed groups of people.
+In order to run a healthy open source community we need to make sure that we triage issues and merge PRs in a timely fashion.
```

Comment:
nitpick: I would prefer `review PRs` rather then `merge PRs` just to set the expectation that not all PRs are to be merged